README.md

```diff
@@ -19,18 +19,42 @@ as `abc` version `X.Y.Z+dune`.
 
 ## Port a new package/version to dune on `opam-overlays`
 
-The typical workflow to port an existing package to dune and publish it on
-`opam-overlays` would be the following:
-
-1. Create a fork of the upstream package `abc`
-2. Port it to dune on a specific branch `dune-universe-vX.Y.Z`, where `X.Y.Z`
-   corresponds to the upstream version you're dunifying. Make sure the opam file
-   has been properly updated and that dune is now part of the package `depends`
-   field and the `dev-repo` and `homepage` point to your fork. It is also
-   recommended to remove the `doc` field to ease the rest of the release
-   process.
-3. Tag it with `X.Y.Z+dune`.
-4. Use `dune-release` to release your port to this repo as described in the
+The typical workflow to port an existing package `pkg`, version `X.Y.Z` to dune
+and publish it on `opam-overlays` would be the following:
+
+1. Create a fork of the upstream package `pkg` repo on dune-universe
+2. Create a branch `dune-universe-vX.Y.Z` pointing to the upstream `X.Y.Z`
+   release tag
+3. Work on the actual port. Make sure the opam file has been properly updated
+   and that dune is now part of the package `depends` field. The build
+   instructions should also be updated to use dune. To ease the release process,
+   the `dev-repo` and `homepage` should point to the dune-universe fork. It is
+   also recommended to remove the `doc` field for similar reasons.
+   Once it is ready, create a PR to merge your port into the
+   `dune-universe-vX.Y.Z` branch of the dune-universe fork.
+4. Once the PR is reviewed and merged, tag the head commit of the
+   `dune-universe-v.X.Y.Z` with `X.Y.Z+dune`.
+5. Use `dune-release` to release the port to this repo as described in the
+   below section.
+
+Note that this requires you to be a member of the `dune-universe` organisation
+to follow all the steps described here. If you'd like to submit a new port as an
+external contributor, please open an issue on this repository with the name of
+the package and the version you wish to port. `dune-universe` maintainers will
+be happy to help you with the admin work so that you can submit the PR described
+in `3`. Once the PR is merged, `dune-universe` maintainers will take care of the
+release.
+
+## Fixing an existing port
+
+If there is an issue with an existing port that we need to fix, we do not remove
+the previous port but release a new one instead. Please follow the steps
```

Comment:
I think it would be good to explain that removing the existing port would break software that depends on it hence we suggest creating new `duneN` releases. Oftentimes I find it easier to adhere to instructions if the reason for them is understandable to me.

Reply:
Did just that!